nsxt/resource_nsxt_policy_project.go

```diff
@@ -143,7 +169,41 @@ func resourceNsxtPolicyProjectPatch(connector client.Connector, d *schema.Resour
 	log.Printf("[INFO] Patching Project with ID %s", id)
 
 	client := infra.NewProjectsClient(connector)
-	return client.Patch(defaultOrgID, id, obj)
+	err := client.Patch(defaultOrgID, id, obj)
+	if err != nil {
+		return err
+	}
+	return patchVpcSecurityProfile(d, connector, id)
```

Comment:
I suggest to wrap this in `HasChange ("default_security_profile")` to avoid unnecessary call